packages/eslint-plugin/tests/rules/unbound-method.test.ts

```diff
@@ -463,7 +463,11 @@ class Foo {
 type foo = new ({ unbound }: Foo) => void;
     `,
     'const { unbound } = { unbound: () => {} };',
-    'function foo({ unbound }: { unbound: () => void } = { unbound: () => {} }) {}',
+    `
+function foo(
+  { unbound }: { unbound: (this: void) => void } = { unbound: () => {} },
```

Comment:
To my understanding, this previously valid test is in fact invalid without a `this: void` parameter type.